com.spacestation14.Launcher.yaml

```diff
@@ -0,0 +1,76 @@
+---
+app-id: com.spacestation14.Launcher
```

Comment:
it is `com` or is it `io` ?

Reply:
It's … a bit odd? My understanding from https://github.com/space-wizards/SS14.Launcher/issues/66#issuecomment-1140082966 is .com will be canonical … eventually. The email domain lives under .com, for instance. Do you think it would be better to assume the current TLD will be around for a long time regardless, given it's pretty well stuck at this point?

Reply:
I should *really* move everything over some time huh...